controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -92,20 +93,36 @@ void client_steering_task::steer_sta()
     }
 
     std::string radio_mac = database.get_node_parent_radio(hostap_mac);
+    auto current_ap_mac   = database.get_node_parent(sta_mac);
 
-    auto request =
-        message_com::create_vs_message<beerocks_message::cACTION_CONTROL_CLIENT_ALLOW_REQUEST>(
-            cmdu_tx, id);
+    // Send 17.1.27	Client Association Control Request
+    if (!cmdu_tx.create(0, ieee1905_1::eMessageType::CLIENT_ASSOCIATION_CONTROL_REQUEST_MESSAGE)) {
+        LOG(ERROR)
+            << "cmdu creation of type CLIENT_ASSOCIATION_CONTROL_REQUEST_MESSAGE, has failed";
+        return;
+    }
 
-    if (request == nullptr) {
-        LOG(ERROR) << "Failed building ACTION_CONTROL_CLIENT_ALLOW_REQUEST message!";
+    auto association_control_request_tlv =
+        cmdu_tx.addClass<wfa_map::tlvClientAssociationControlRequest>();
+    if (!association_control_request_tlv) {
+        LOG(ERROR) << "addClass wfa_map::tlvClientAssociationControlRequest failed";
         return;
     }
-    request->mac()  = network_utils::mac_from_string(sta_mac);
-    request->ipv4() = network_utils::ipv4_from_string(database.get_node_ipv4(sta_mac));
-    Socket *sd      = database.get_node_socket(radio_mac);
-    TASK_LOG(DEBUG) << "sending allow request for " << sta_mac << " to " << radio_mac
-                    << " id=" << int(id);
+
+    association_control_request_tlv->alloc_sta_list();
+    auto sta_list         = association_control_request_tlv->sta_list(0);
+    std::get<1>(sta_list) = network_utils::mac_from_string(sta_mac);
+    association_control_request_tlv->bssid_to_block_client() =
+        network_utils::mac_from_string(current_ap_mac);
+    association_control_request_tlv->validity_period_sec() = disassoc_timer;
+    association_control_request_tlv->association_control() =
+        wfa_map::tlvClientAssociationControlRequest::BLOCK;
```

Comment:
This  should be an UNBLOCK. The blocking is done below (cACTION_CONTROL_CLIENT_DISALLOW_REQUEST). That one should also be updated.